man/effectAllele.Rd

```diff
@@ -53,7 +53,7 @@ assoc <- assocTestSingle(iterator, nullmod)
 
 # add effect allele to the results
 eff <- effectAllele(seqData, variant.id=assoc$variant.id)
-assoc <- dplyr::left_join(assoc,eff)
+assoc <- merge(assoc, eff)
```

Comment:
I think `merge` doesn't preserve order while `left_join` does. Will this reorder `assoc`?  Does it matter?

Reply:
It won't in this case, because it's merging on variant.id and assoc is already in order by variant.id. This is just an example so I don't think it matters. In practice I'd want to use left_join, but I don't want to leave dplyr in Suggests for just this single example. I just had an idea, though - I could comment out that line so the dependency is not there but users can still see it.

Reply:
Changing my mind yet again... we already have ggplot2 in Suggests so I might as well include dplyr there as well.